standardization/src/test/scala/za/co/absa/enceladus/standardization/samples/TestSamples.scala

```diff
@@ -74,13 +76,13 @@ object TestSamples {
     new StdEmployee(name = "John0", surname = "Unknown Surname", hoursWorked = Some(List(8, 7, 8, 9, 12, 0)),
         employeeNumbers = List(EmployeeNumberStd("SAP", List(456, 123)), EmployeeNumberStd("WD", List(5))), startDate =  new java.sql.Date(startDate), errCol = List(ErrorMessage.stdNullErr("surname"), ErrorMessage.stdNullErr("hoursWorked[*]"))),
     new StdEmployee(name = "John1", surname = "Doe1", hoursWorked = Some(List(99, 99, 76, 12, 12, 24)), startDate = new java.sql.Date(0), errCol = List(ErrorMessage.stdCastErr("startDate", "Two Thousand Something"))),
-    new StdEmployee(name = "John2", surname = "Unknown Surname", hoursWorked = None, startDate = new java.sql.Date(startDate), updated = Some(new java.sql.Timestamp(1437053544l *1000l)), errCol = List(ErrorMessage.stdNullErr("surname"), ErrorMessage.stdNullErr("hoursWorked"))),
-    new StdEmployee(name = "John3", surname = "Unknown Surname", hoursWorked = Some(List()), startDate = new java.sql.Date(startDate), updated = Some(new java.sql.Timestamp(1437042744l * 1000l)), errCol = List(ErrorMessage.stdNullErr("surname"))))
+    new StdEmployee(name = "John2", surname = "Unknown Surname", hoursWorked = None, startDate = new java.sql.Date(startDate), updated = Some(new java.sql.Timestamp(1437053544000l)), errCol = List(ErrorMessage.stdNullErr("surname"), ErrorMessage.stdNullErr("hoursWorked"))),
```

Comment:
I see you replaced one magical constant for two. Why not to use _Timestamp.valueOf()_ instead (https://docs.oracle.com/javase/6/docs/api/java/sql/Timestamp.html#valueOf(java.lang.String))?

Reply:
yes, it make sense.